src/test/groovy/com/synacy/gradprogram/spock/exercise/DeliveryServiceSpec.groovy

```diff
@@ -0,0 +1,50 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+
+class DeliveryServiceSpec extends Specification {
+
+    DeliveryService service
+
+    DateUtils dateUtils = Mock(DateUtils)
+    DeliveryRequestRepository deliveryRequestRepository = Mock(DeliveryRequestRepository)
+
+    def setup() {
+        service = new DeliveryService(dateUtils, deliveryRequestRepository)
+    }
+
+    def "createDelivery should be able to save the created delivery request."(){
```

Comment:
We can improve the test by asserting the Courier value depending on the total cost of the order.